tests/errors/codegen/fortran/FORTRAN_ALLOCATABLE_IN_EXPRESSION.py

```diff
@@ -1,3 +1,3 @@
 # pylint: disable=missing-function-docstring, missing-module-docstring/
 import numpy as np
-a = np.ones(4)*5
+a = np.empty(4)*5
```

Comment:
why did you change this example ?

Reply:
Now that the allocation has been separated from the printing `ones` is printed as `1` in fortran. So:
```
a = np.ones(4)*5
```
is correctly translated to:
```
allocate(a(0:4))
a = 1*5
```

Reply:
ah okay , perfect

Reply:
As all the values of the allocatable in the expression are the same we don't need a temporary array so there is no problem